### PR TITLE
Redesigned card

### LINKED ISSUE #1638: Frontend UI used for the cards image is not up-to the mark in Map Gallery
### Problem
->Frontend UI used for the cards image can be improved

Screenshot 2022-04-12 122602

### Relevant URLs

https://mapknitter.org/gallery

### PR DESCRIPTION
**Pull request by asaayushisingh**
Fixes #1638  (<=== Add issue number here)

 ### Before 

Screenshot 2022-04-12 122602 

### After 

redesigned

 ### Note 
As we don't have the access to the database that's why we are not able to fetch the live data. 

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/mapknitter-reviewers` for help, in a comment below

### CODE REVIEW

**Comment by asaayushisingh** [comment]
@TildaDares @jywarren I'd appreciate it if I could get this pull request reviewed, Thank you!.

**Comment by on2onyekachi** [comment]
Hi @asaayushisingh, This is a good fix! on the images. Considering the contribution workflow, a mentor @jywarren @TildaDares @cesswairimu  will have to review the issue raised before this PR can be merged!. 

So, it's much better to get a review on an issue before creating a PR for it. Likewise the other.

**Comment by asaayushisingh** [comment]
> Hi @asaayushisingh, This is a good fix! on the images. Considering the contribution workflow, a mentor @jywarren @TildaDares @cesswairimu will have to review the issue raised before this PR can be merged!.
> 
> So, it's much better to get a review on an issue before creating a PR for it. Likewise the other.



> Hi @asaayushisingh, This is a good fix! on the images. Considering the contribution workflow, a mentor @jywarren @TildaDares @cesswairimu will have to review the issue raised before this PR can be merged!.
> 
> So, it's much better to get a review on an issue before creating a PR for it. Likewise the other.

Yeah Sure! Thank You!

**Comment by asaayushisingh** [comment]
Hey @cesswairimu! Kindly review this. Thank You!

**Comment by cesswairimu** [comment]
Fantastic :tada: @asaayushisingh  These are lovely! thanks for working on this. Could you pleasse mark your pull request as readyy for review if you are done with the changes. Thanks

**Comment by asaayushisingh** [comment]
> Fantastic 🎉 @asaayushisingh These are lovely! thanks for working on this. Could you pleasse mark your pull request as readyy for review if you are done with the changes. Thanks


@cesswairimu Thanks a lot!..Yeah, I marked it ready for review.

File: app/assets/stylesheets/style.scss [hunk 1]
```diff
@@ -62,7 +62,17 @@ It was originally created by Caroline Hadilaksono: http://www.hadilaksono.com
 .map-list div:nth-child(4n+1){
   clear: left;
 }
-
+.e2_48 {
```
Comment (cesswairimu) [inline app/assets/stylesheets/style.scss:65]: Hi, could we use a better class name for this? not important but better for people looking at the file to figure out where its used

File: app/views/front_ui/_maps.html.erb [hunk 2]
```diff
@@ -9,27 +9,27 @@
   <% maps.each do |map| %>
       <% warp = map.placed_warpables.first %>
     <div class="card-deck map col-md-6 col-lg-3 col-xs-12 mb-2 ml-2">
-        <div class="card bg-transparent card-without-border">
+        <div class="card bg-transparent card-without-border ">
         <a href="/maps/<%= map.slug %>">
             <% if warp.present? %>
             <img class="card-img-top map-img" src="<%= warp.image.url  %>" alt="<%= map.name %>">
```
Comment (cesswairimu) [inline app/views/front_ui/_maps.html.erb:15]: This is the image displayed when a map has an image, kindly check if the two will have the same feel as the default one.

File: app/assets/stylesheets/style.scss [hunk 3]
```diff
@@ -62,7 +62,17 @@ It was originally created by Caroline Hadilaksono: http://www.hadilaksono.com
...
   clear: left;
 }
-
+.e2_48 { 
+	width:250.7894592285156px;
+	height:160.3662719726562px;
+	/* position:absolute; */
+	left:-0.00002248663622594904px;
+	top:-88.56105041503906px;
+	background-image:url("https://3000-publiclab-mapknitter-1f5i6spevl6.ws-us39.gitpod.io/assets/image-cd28b9101644e1fb24be3adb6baf2d8717d366c3df526f4ec8436b7efba7a73c.png");
```
Comment (cesswairimu) [inline app/assets/stylesheets/style.scss:71]: Kindly check if the bg-image url will work remotely.

**Review by cesswairimu — CHANGES_REQUESTED** [review]
Thanks  for working on this.

File: app/assets/stylesheets/style.scss (hunk 3 above)
Comment (asaayushisingh) [reply app/assets/stylesheets/style.scss:71]: @cesswairimu Thanks for pointing it out! 💯  I had made the required changes, now the bg-image URL will work remotely 😄. Kindly review it. Thank You!

File: app/assets/stylesheets/style.scss (hunk 1 above)
Comment (asaayushisingh) [reply app/assets/stylesheets/style.scss]: Yeah! That will be great 😃

Comment (asaayushisingh) [reply app/assets/stylesheets/style.scss]: @cesswairimu  I had made the required changes. Kindly review it! Thank You!

File: app/views/front_ui/_maps.html.erb (hunk 2 above)
Comment (asaayushisingh) [reply app/views/front_ui/_maps.html.erb]: @cesswairimu I had made the changes in the conditional statement. Kindly review it ! Thank You!

**Review by cesswairimu — APPROVED** [review]

**Comment by cesswairimu** [comment]
Thanks

**Comment by asaayushisingh** [comment]
> Thanks

Thank You @cesswairimu  😄.  @on2onyekachi  Thank You for the feedback 😄

**Pull request merged**